dev_docs/cxg.md

```diff
@@ -1,5 +1,3 @@
-## UPDATE (9/30/2020): Starting today, the name Corpora will only be used as the internal project name, with cellxgene Data Portal being the official product name
-
 # CXG Data Format Specification
 
 Document Status: _draft_
```

Comment:
given the handful of MUST/SHOULD changes, recommend we bump the CXG version number (at least to 0.2.1)

Reply:
Happy to bump it, but in fact, I was only attempting to update the doc to match what appeared to be the "intended" 0.2.0 specification, rather than trying to change the specification.  I was considering the h5ad->cxg _code_ to be the ground truth for what constituted the spec.  And as this specification is for our private use only (right?), making changes to the text for an _existing_ version of the spec seemed okay. But I can agree that any changes to the doc should reasonably result in a version bump.

Reply:
Not going to bump the version, since this 0.2.0 spec defines the current state of affairs, in terms of real data in the wild. There are files that now have 0.2.0 cxg version and conform to this spec as stated. We'll need to be more careful in the future. But honestly, all of the substantive changes are in the `corpora` metadata, which is not even part of the CXG spec technically.

Reply:
works for me